packages/frontend/editor-ui/src/types/canvas.ts

```diff
@@ -125,7 +125,7 @@ export interface CanvasNodeData {
 		waitingForNext?: boolean;
 	};
 	runData: {
-		outputMap: ExecutionOutputMap;
+		outputMap?: ExecutionOutputMap;
```

Comment:
Not sure what changed that's causing this but unlikely related to any builder changes itself, more likely canvas related.